content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -847,6 +847,35 @@ not vulnerable to ordering changes in the list.
 
 Once the last finalizer is removed, the resource is actually removed from etcd.
 
+### Force deletion
+
+{{< feature-state feature_gate_name="AllowUnsafeMalformedObjectDeletion" >}}
+
+{{< caution >}}
+This may break the workload associated with the resource being force deleted, if it
+relies on the normal deletion flow, so cluster breaking consequences may apply.
+{{< /caution >}}
+
+By enabling the delete option `ignoreStoreReadErrorWithClusterBreakingPotential`, the
+user can perform an unsafe, and force **delete** operation of an undecryptable/corrupt
+resource. This option is behind a new ALPHA feature gate, and it is disabled by
+default. In order to use this option, the cluster operator must enable the feature by
+setting the command line option `--feature-gates=AllowUnsafeMalformedObjectDeletion=true`.
+
+{{< note >}}
+The user performing the force **delete** operation must have the
+`unsafe-delete-ignore-read-errors` permission on the resource.
+{{< /note >}}
+
+A resource is considered corrupt if it can not be successfully retrieved from the
+storage due to a) transformation error e.g. decryption failure, or b) the object
+failed to decode. Normal deletion flow is attempted first, and if it fails with
+a `corrupt resource` error then it triggers the unsafe, and force delete. A force
+**delete** operation ignores finalizer constraints, and skips precondition checks.
+
```

Comment:
```suggestion
Am I right that you cannot use `ignoreStoreReadErrorWithClusterBreakingPotential` to skip finalizers if the backing data are readable and could be removed without issue?

I think that's the case from what's written here, but I wanted to check.
```

Reply:
yes, you cannot avoid finalizers by setting the `ignoreStoreReadErrorWithClusterBreakingPotential` option, the apiserver will throw an error, I added a note.